tests/unit/src/kakarot/precompiles/test_precompiles.py

```diff
@@ -0,0 +1,29 @@
+import random
+
+import pytest
+import pytest_asyncio
+from starkware.starknet.testing.contract import StarknetContract
+from starkware.starknet.testing.starknet import Starknet
+
+
+@pytest_asyncio.fixture(scope="module")
+async def precompiles(starknet: Starknet):
+    return await starknet.deploy(
+        source="./tests/unit/src/kakarot/precompiles/test_precompiles.cairo",
+        cairo_path=["src"],
+        disable_hint_validation=True,
+    )
+
+
+@pytest.mark.asyncio
+class TestPrecompiles:
+    async def test_precompiles(self, precompiles):
```

Comment:
it's better to name this func the same as the one in the corresponding cairo file `precompiles_should_throw_on_not_implemented` for log reason